designer/server/src/routes/forms/library.js

```diff
@@ -125,7 +141,17 @@ export default [
 ]
 
 /**
- * @typedef {Request<{ Query: PaginationOptions }>} RequestFormsLibrary
+ * @typedef {{
+ *   sort?: 'updated_desc' | 'updated_asc' | 'title_asc' | 'title_desc'
+ * }} SortOptions
```

Comment:
Mind if we keep the older syntax with descriptions?

Probably not for our team currently, but IntelliJ might not do multi-line expanded JSDoc

```suggestion
 * @typedef {object} SortOptions
 * @property {'updated_desc' | 'updated_asc' | 'title_asc' | 'title_desc'} [sort] - The sort order
```